api/bases/ansibleee.openstack.org_openstackansibleees.yaml

```diff
@@ -1904,6 +1907,12 @@ spec:
                           items:
                             type: string
                           type: array
+                        vars:
+                          items:
+                            type: object
+                            x-kubernetes-preserve-unknown-fields: true
+                          type: array
```

Comment:
`ansibleVars` in dataplane-operator node was converted to a string recently [1]. Should this `vars` field be consistent with it?

[1] 
https://github.com/openstack-k8s-operators/dataplane-operator/pull/23/commits/fa4d55e520360a3cbeeaa38cbeca317b400d3cb8#diff-1988fd56b05f07f65f48a21b48d923dcd18cb5d992d6dc1a0fba805f9e4cf553L54

Reply:
Nice catch! I'm gonna change it!